src/client/components/network-editor/index.js

```diff
@@ -148,15 +149,12 @@ export class NetworkEditor extends Component {
     });
   }
 
+  getMinQValue() {
+    return this.cy.nodes().min(n => n.data('padj')).value;
+  }
+
   getMaxQValue() {
-    return this.cy.nodes().reduce(
-      (prevVal, node) => {
-        if(typeof prevVal === 'undefined')
-          return 0;
-        const qval = node.data('padj');
-        return qval ? Math.max(qval, prevVal) : prevVal;
-      }
-    );
+    return this.cy.nodes().max(n => n.data('padj')).value;
```

Comment:
FYI:  Cytoscape has `min()` and `max()` methods just like arrays do, which is more convenient that `reduce()`.

Reply:
Or if you want to use an array method that Cytoscape doesn't have, you can do `eles.toArray().whatever()...`